core/extraction/round_1.py

```diff
@@ -0,0 +1,208 @@
+import pandas as pd
+
+
+def ingest_round_1_data(df_map: pd.DataFrame) -> dict[pd.DataFrame]:
+    """
+    Extract data from Round 1 Reporting Template into column headed Pandas DataFrames.
+
+    :param df_ingest: DataFrame of parsed Excel data.
+    :return: Dictionary of extracted "tables" as DataFrames.
+    """
+    df_dictionary = dict()
+
+    df_dictionary["programme_outcomes"] = extract_programme_outcomes(df_map["programme_outcomes"])
```

Comment:
ah, so I've just realised, that since we don't actually need to do any manipulation to get these sheets in, and we are just naming them the same thing they are already called on the tabs, we don't need to run any individual extraction methods at all. pd.read_excel already does all that. Sorry. 

Effectively, input param df_map is the same as df_dictionary in this function.

However, what we have is probably useful as a placeholder structure for when we do some transformation, so it can be left in the current format for now :)